spec/haskell/src/SEL4/Object/Structures/AARCH64.hs

```diff
@@ -70,6 +70,11 @@ atcbContextSet uc atcb = atcb { atcbContext = uc }
 atcbContextGet :: ArchTCB -> UserContext
 atcbContextGet = atcbContext
 
+data ArchTcbFlag = FpuDisabled
```

Comment:
in some cases when we have datatypes that are boringly similar to what we have in ASpec, we don't translate them and instead introduce an alias in the skeleton files; not sure if that's useful here

Reply:
I think that would make sense, also for any future flags. Not sure of there we have a good location to put these in for arch specific data types, but it's worth looking.

Reply:
What's the outcome on this @corlewis ?

Reply:
The abstract and exec specs now share this datatype, along with `archTcbFlagToWord` and the generic equivalents.

Reply:
Very nice. I like this approach, and it made sense to make a generic Structs_B